src/site/_includes/components/header/header.scss

```diff
@@ -0,0 +1,106 @@
+.main-menu {
+  background-color: $color-yellow;
+  transition: height 250ms, box-shadow 250ms;
+  box-shadow: 0 0 0 #00000000;
+
+  & > div {
+    border-bottom: 2px solid $color-darkyellow;
+  }
+
+  &__inner {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .logo {
+    margin-right: 2em;
+    a {
+      display: block;
+    }
+  }
+
+  button {
+    display: none;
+  }
+
+  svg {
+    max-width: 60%;
```

Comment:
Also, 60% is super arbitrair, zou je me kunnen uitleggen waarom dit zo is? En zou je er een variabele als `my-specific-variable-name-because-60-percent` ofzo.